controllers/hyperconverged/hyperconverged_controller.go

```diff
@@ -1045,6 +1055,34 @@ func (r *ReconcileHyperConverged) detectTaintedConfiguration(req *common.HcoRequ
 	}
 }
 
+func (r *ReconcileHyperConverged) getSystemHealthStatus(req *common.HcoRequest) string {
+	isConditionAvailableTrue := req.Conditions.IsStatusConditionTrue(hcov1beta1.ConditionAvailable)
+	isConditionDegradedTrue := req.Conditions.IsStatusConditionTrue(hcov1beta1.ConditionDegraded)
+	isConditionProgressingTrue := req.Conditions.IsStatusConditionTrue(hcov1beta1.ConditionProgressing)
+	isConditionReconcileCompleteTrue := req.Conditions.IsStatusConditionTrue(hcov1beta1.ConditionReconcileComplete)
```

Comment:
each one of these variables is used only once. Not sure we need them, or to call `IsStatusConditionTrue` instead.